Bricks/util/iterator.h

```diff
@@ -49,7 +49,8 @@ struct GenericMapIterator final {
 
 template <typename MAP>
 struct GenericMapAccessor final {
-  using iterator_t = GenericMapIterator<MAP>;
+  using const_iterator = GenericMapIterator<MAP>;
+  using iterator_t = const_iterator;
```

Comment:
Same as above -- suggest we leave renaming to the last pass, added a DIMA_FIXME to grep for it later.